### PR TITLE
Dot Net Assignment 2 Completed

### PR DESCRIPTION
**Pull request by emtec-damodhar-jadhao**
Created a calculator using .Net Core WEB API and Class Library

### CODE REVIEW

File: WebAPICore/Controllers/WeatherForecastController.cs [hunk 1]
```diff
@@ -0,0 +1,33 @@
+using Microsoft.AspNetCore.Mvc;
+
+namespace WebAPICore.Controllers
+{
+    [ApiController]
+    [Route("[controller]")]
+    public class WeatherForecastController : ControllerBase
```
Comment (nitish-poojari) [inline WebAPICore/Controllers/WeatherForecastController.cs]: remove autogenerated controller if not required

File: Calculator_Library/Class1.cs [hunk 2]
```diff
@@ -0,0 +1,25 @@
+﻿namespace Calculator_Library
+{
+    public class Class1
```
Comment (nitish-poojari) [inline Calculator_Library/Class1.cs]: Give proper name to class

File: WebAPICore/Controllers/WeatherForecastController.cs (hunk 1 above)
Comment (emtec-damodhar-jadhao) [reply WebAPICore/Controllers/WeatherForecastController.cs]: auto generated Controller deleted

File: Calculator_Library/Class1.cs (hunk 2 above)
Comment (emtec-damodhar-jadhao) [reply Calculator_Library/Class1.cs]: class name updated

**Review by nitish-poojari — APPROVED** [review]

**Pull request merged**